web/src/lib/utils/cases-helpers.test.ts

```diff
@@ -41,4 +42,12 @@ describe("cases-helpers", () => {
       expect(camelCaseToKebabCase("businessAddressLine1")).toEqual("business-address-line1");
     });
   });
+
+  describe("capitalizeEachWord", () => {
+    it("capitalizes first letter of each word", () => {
+      expect(capitalizeEachWord("business address")).toEqual("Business Address");
+      expect(capitalizeEachWord("business Address")).toEqual("Business Address");
+      expect(capitalizeEachWord("Business address")).toEqual("Business Address");
```

Comment:
[pebble] could have the `expect(capitalizeEachWord("Business Address")).toEqual("Business Address");` case as well

Reply:
done